usr.bin/lsvfs/lsvfs.c

```diff
@@ -44,6 +46,18 @@ main(int argc, char **argv)
 
 	argc--, argv++;
 
+	if (sysctlbyname("vfs.conflist", NULL, &buflen, NULL, 0) < 0)
```

Comment:
I was about to commit this change, when I have noticed one more thing. Sorry about that.

Do we want to always fetch the full list.
When we do:
```
lsvfs ufs
```

We don't need to fetch the `vfs.conflist` to check for UFS fs. Now we are adding a small overhead to the utility that we always fetch the list. @kfv what do you think?

Reply:
I’ll look into this in more detail and get back to you shortly.